packages/website/docs/core/inputs_transform/TextSplitter.md

```diff
@@ -0,0 +1,83 @@
+## TextSplitter
+
+The `TextSplitter` is a text splitting callable event designed to split a given text into multiple contexts based on specified parameters. It extends the functionality of ContextSplitter by allowing the text to be split using a custom separator and controlling whether the separator should be retained in the outputs.
```

Comment:
I think the sentence "It extends the functionality of ContextSplitter..." is not very helpful for users to understand the usage of TextSplitter. Maybe focusing on why TextSplitter should be used is more helpful